src/styles/variables.css

```diff
@@ -1,65 +1,106 @@
+
+/* !!! REMOVED --action-secondary: rgb(76, 93, 255), replaced with colony-blue */
+
+/* !!! REMOVED --violet: rgb(89, 100, 240), replaced with colony-blue
+  it was used only in `PayoutsList`, and that is not used anywhere for now */
+
+/* !!! REMOVED --grey-darkrgb(47, 47, 47), replaced with --dark; not used in many places */
+
+/* !!! REMOVED temp-grey-0: rgb(195, 204, 205), replaced with --text-disabled; almost identical */
+
+/* !!! REMOVED --grey-3: rgb(82, 84, 85), replaced with --dark, not used much, looks similar */
+
+/* !!! REMOVED
+    --grey-2: rgb(54, 54, 54);
+    --temp-grey-2: rgb(46, 50, 53);
+    --temp-grey-8: rgb(52, 55, 56);
+    replaced with --dark-grey, not used that much, looks very similar */
+
+/* !!! REMOVED
+    --temp-grey-11: rgb(136, 136, 136);
+    --temp-grey-4: rgb(153, 153, 153);
+    replaced with --dark-grey, not used that much, looks very similar */
+
+/* !!! REMOVED
+  --grey-blue-0: rgb(232, 232, 232);
+  --grey-blue-1: rgb(232, 236, 245);
+  --grey-blue-2: rgb(244, 246, 250);
+  replaced with --light-grey-blue which is combination of all 3 of them (via Raul's function) */
+
+/* !!! REMOVED --grey-5: rgb(173, 186, 189), replaced with --text-disabled, not used that much, looks similar */
+
+/* !!! REMOVED --temp-grey-blue-8: rgb(238, 242, 245), replaced with --colony-light-blue, duplicate */
+
+/* !!! REMOVED --temp-grey-6: rgb(220, 226, 227), not used anywhere */
+
+/* !!! REMOVED --temp-grey-blue-2: rgb(209, 214, 215), replaced with --temp-grey-5, almost the same */
+
+/* !!! REMOVED --temp-grey-blue-1: rgb(242, 246, 246), not used anywhere */
+
+/* !!! REMOVED  --temp-grey-blue-3: rgb(251, 252, 253), not used anywhere */
+
+/* !!! REMOVED  --temp-grey-blue-12: rgb(152, 164, 193), only found it in an unused class in Members.css */
+
 :root {
   --primary: rgb(104, 210, 210); /* curreny colony primary */
-  --secondary: rgb(0, 230, 196); /* greenish primary color */
   --colony-white: rgb(251, 251, 251);
   --colony-black: rgb(31, 35, 39);
   --colony-blue: rgb(79, 93, 255);
-  --colony-light-blue: rgb(238, 242, 245);
   --dark-blue: rgb(69, 85, 230);
   --sky-blue: rgb(90, 145, 255);
   --danger: rgb(248, 43, 101); /* user mentions color */
-  --violet: rgb(89, 100, 240);
   --pink: rgb(254, 94, 124);
   --golden: rgba(255, 176, 0); /* Think pending states (golden yellow) */
   --dark: rgb(60, 68, 77); /* Used for colony name display (colony simplified) */
-  --grey-dark: rgb(47, 47, 47);
-  --grey-1: rgb(58, 58, 58);
-  --grey-2: rgb(54, 54, 54);
-  --grey-3: rgb(82, 84, 85);
-  --grey-4: rgb(124, 124, 124);
-  --grey-5: rgb(173, 186, 189);
-  --grey-blue-0: rgb(232, 232, 232);
-  --grey-blue-1: rgb(232, 236, 245);
-  --grey-blue-2: rgb(244, 246, 250);
-  --grey-blue-3: rgb(246, 246, 246);
-  --action-secondary: rgb(76, 93, 255);
+  --text-disabled: rgb(194, 204, 204);
+  --colony-light-blue: rgb(238, 242, 245);
+  --light-grey-blue: rgb(236, 238, 242);
+  --drop-shadow: rgba(62, 118, 244, 0.14);
```

Comment:
I feel a bit "iffy" having this color name which describes an object/attribute rather than an actual color.

But since it's only being used for shadows, and since we're going to change it anyway with the new design, it can stay in.